Intersect (Core)/GameObjects/Annotations/EditorDictionaryAttribute.cs

```diff
@@ -102,6 +102,19 @@ public override string Format(Type stringsType, object value)
             }
         }
 
+        public string[] GetNames(Type stringsType)
```

Comment:
Just confirming, this _is_ still used, correct?

Reply:
Nope, removed